backend/canisters/identity/impl/src/lib.rs

```diff
@@ -184,6 +194,8 @@ struct Data {
     rng_seed: [u8; 32],
     challenges: Challenges,
     test_mode: bool,
+    #[serde(default)]
+    account_linking_codes: HashMap<String, AccountLinkingCode>,
```

Comment:
It's probably worth putting this in its own struct so that we can expose helper functions, and we'll likely want additional fields in the future too such as last_updated, etc.